site/templates/bases/base.tera.html

```diff
@@ -8,7 +8,7 @@
         <title>{% block title %}Hipcheck{% endblock %}</title>
         {% include "partials/head.tera.html" %}
     </head>
-    <body class="bg-white dark:bg-neutral-900 {% block body_classes %}{% endblock %}">
+    <body class="bg-white dark:bg-neutral-900 {% block body_classes %}{% endblock %} px-4 md:px-8 lg:px-16">
```

Comment:
Hm, I don't think this is quite the right fix. If you look at the homepage, you'll see the navigation bar at the top is squeezed inward horizontally, and the blue "values" section, which previously spanned the width of the page, is now boxed in. I think these padding styles should only be applied to the section with the intro text and diagram.